src/xinterpreter.cpp

```diff
@@ -237,7 +237,7 @@ __get_cxx_version ()
             /*
                 // Publish a mime bundle for the last return value if
                 // the semicolon was omitted.
-                if (!silent && output.hasValue() && trim(code).back() != ';')
+                if (!config.silent && output.hasValue() && trim(code).back() != ';')
```

Comment:
Should we uncomment this block or drop it? (I know it is not related to this PR, but I don't remember why we have it).